cczoo/common/docker/gramine/anolisos-gramine-sgx-dev.dockerfile

```diff
@@ -0,0 +1,99 @@
+
+# Copyright (c) 2022 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# https://github.com/oscarlab/graphene/blob/master/Tools/gsc/images/graphene_aks.latest.dockerfile
+FROM openanolis/anolisos:8.4-x86_64 AS Anolisos
+
+ENV INSTALL_PREFIX=/usr/local
+ENV LD_LIBRARY_PATH=${INSTALL_PREFIX}/lib:${INSTALL_PREFIX}/lib64:${LD_LIBRARY_PATH}
+ENV PATH=${INSTALL_PREFIX}/bin:${LD_LIBRARY_PATH}:${PATH}
+# Add steps here to set up dependencies
+RUN yum install -y \
+    openssl-devel \
+    libcurl-devel \
+    protobuf-devel \
+    yum-utils.noarch \
+    python3 \
+    wget
+
+# Intel SGX
+RUN mkdir /opt/intel && cd /opt/intel \
+    && wget https://mirrors.openanolis.cn/inclavare-containers/bin/anolis8.4/sgx-2.15.1/sgx_rpm_local_repo.tar.gz 
+RUN cd /opt/intel && sha256sum sgx_rpm_local_repo.tar.gz \
+    && tar xvf sgx_rpm_local_repo.tar.gz \
+    && yum-config-manager --add-repo file:///opt/intel/sgx_rpm_local_repo \
+    && yum --nogpgcheck install -y libsgx-urts libsgx-launch libsgx-epid libsgx-quote-ex libsgx-dcap-ql libsgx-uae-service libsgx-dcap-quote-verify-devel 
+RUN yum groupinstall -y 'Development Tools'
+
+# COPY patches/libsgx_dcap_quoteverify.so  /usr/lib64/
+RUN yum install -y --nogpgcheck sgx-dcap-pccs libsgx-dcap-default-qpl
+
+# Gramine
+ENV GRAMINEDIR=/gramine
+ENV SGX_DCAP_VERSION=DCAP_1.11
+ENV GRAMINE_VERSION=v1.2
+ENV ISGX_DRIVER_PATH=${GRAMINEDIR}/driver
+ENV PKG_CONFIG_PATH=/usr/local/lib64/pkgconfig/
+ENV LC_ALL=C.UTF-8 LANG=C.UTF-8
+ENV WERROR=1
+ENV SGX=1
+ENV GRAMINE_PKGLIBDIR=/usr/local/lib64/gramine
+ENV ARCH_LIBDIR=/lib64
+
+RUN yum install -y gawk bison python3-click python3-jinja2 golang ninja-build 
+RUN yum install -y openssl-devel protobuf-c-devel python3-protobuf protobuf-c-compiler
+RUN yum install -y gmp-devel mpfr-devel libmpc-devel isl-devel nasm python3-devel mailcap
+
+RUN ln -s /usr/bin/python3 /usr/bin/python \
+    && pip3 install --upgrade pip \
```

Comment:
Please reference https://github.com/docker-library/docs/tree/master/python#how-to-use-this-image to learn how to use pip3 install in dockerfile.